src/bp/core/config/botpress.config.ts

```diff
@@ -97,7 +97,7 @@ export type BotpressConfig = {
      */
     backlog: number
     /**
-     * @default 100kb
```

Comment:
I don't think you meant to commit that?  If yes that's *way* to big

Reply:
I simply merged master ... @epaminond  did you merge this somewhere ? I can revert the changes if necessary